LibreHardwareMonitor/UI/Themes/ThemedHScrollIndicator.cs

```diff
@@ -39,6 +42,57 @@ public ThemedHScrollIndicator(HScrollBar scrollBar)
             scrollBar.ValueChanged += (s, e) => Invalidate();
 
             scrollBar.Height = 0;
+            this.MouseDown += OnMouseDown;
+        }
+
+        private void OnMouseDown(object sender, MouseEventArgs e)
+        {
+            if (_isScrolling)
```

Comment:
Can you reformat both changed files please? Thanks :)

Reply:
reformat depend on the editorconfig rules.